src/renderer/components/cluster-manager/cluster-actions.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { clusterSettingsURL } from "../+cluster-settings";
+import { landingURL } from "../+landing-page";
+
+import { clusterStore } from "../../../common/cluster-store";
+import { broadcastMessage, requestMain } from "../../../common/ipc";
+import { clusterDisconnectHandler } from "../../../common/cluster-ipc";
+import { ConfirmDialog } from "../confirm-dialog";
+import { Cluster } from "../../../main/cluster";
+
+const navigate = (route: string) =>
+  broadcastMessage("renderer:navigate", route);
+
+const deactivateCluster = (clusterId: string) => {
+  if (clusterStore.isActive(clusterId)) {
```

Comment:
This deactivation can be moved to `ClusterStore`.

Reply:
Isn't this code already in `ClusterStore`?

Reply:
I guess it isn't in `ClusterStore` but I agree with @aleksfront that it should be.

Reply:
It wasn't, moved it over there